qadence/backend.py

```diff
@@ -168,8 +168,9 @@ def observable(self, observable: AbstractBlock, n_qubits: int) -> ConvertedObser
     def convert(
         self, circuit: QuantumCircuit, observable: list[AbstractBlock] | AbstractBlock | None = None
     ) -> Converted:
-        """Convert an abstract circuit (and optionally and observable) to their native
-        representation. Additionally, this function constructs an embedding function which maps from
+        """Convert an abstract circuit and an optional observable to their native representation.
```

Comment:
Why here the docstring was shortened?

Reply:
Well, it's been broken into few lines to comply with the docstring formatter. The first line has been shortened just to comply with the line length.